e2e/txpool_test.go

```diff
@@ -277,13 +273,28 @@ func TestTxPool_TransactionCoalescing(t *testing.T) {
 	// Add the transaction with the gap nonce value
 	addReq := generateReq(1)
 
-	_, addErr := clt.AddTxn(context.Background(), addReq)
+	addResp, addErr := clt.AddTxn(context.Background(), addReq)
```

Comment:
Why don't you set timeout?

Reply:
Forgot 🤦‍♂️ 

Fixed 👍 

[cb32a27](https://github.com/0xPolygon/polygon-edge/pull/485/commits/cb32a270372f1812ec401c110d88ceb81bccdeca)